src/serac/physics/solid_mechanics.hpp

```diff
@@ -196,7 +196,11 @@ class SolidMechanics<order, dim, Parameters<parameter_space...>, std::integer_se
     // to be the displacement
     auto* amg_prec = dynamic_cast<mfem::HypreBoomerAMG*>(nonlin_solver_->preconditioner());
     if (amg_prec) {
-      amg_prec->SetElasticityOptions(&displacement_.space());
+      // amg_prec->SetElasticityOptions(&displacement_.space());
+
+      // The above call was seg faulting in Hypre in the HYPRE_BoomerAMGSetInterpRefine(amg_precond, interp_refine)
+      // method as of version v2.26.0. Instead, we just set the system size for Hypre.
+      amg_prec->SetSystemsOptions(dim, true);
```

Comment:
Is this change equivalent to the old line, or is it a temporary workaround while the segfault is figured out?

Reply:
Temporary. I will add a TODO.